charts/knative-serving-istio/templates/istio/release-manifests.yaml

```diff
@@ -0,0 +1,174 @@
+# The manifests declared here is a hack to get around the inconsistent use of .Release.namespace and .Values.global.istioNamespace.
+# Release namespace is not known untill runtime, but some resources are using  .Values.global.istioNamespace
+# eg https://github.com/istio/istio/blob/e7911b20b131082ff2f8e648735692036c814d69/manif[…]charts/istio-control/istio-discovery/templates/rolebinding.yaml vs
+# https://github.com/istio/istio/blob/9ff1e2a02722f013d5ec063bd27459a78acef711/manif[…]/charts/istio-control/istio-discovery/templates/deployment.yaml
+# To get around this, we declare the same manifests here but replace .Values.global.istioNamespace with {{ .Release.namespace }}
```

Comment:
I know this is a workaround, but do you have any suggestions for how will we keep these resources in sync when there are updates on istio templates?

Reply:
I have deleted this entire folder also.